src/main/kotlin/frc/team5499/frc2018Kotlin/auto/actions/DriveStraightAction.kt

```diff
@@ -0,0 +1,38 @@
+package frc.team5499.frc2018Kotlin.auto.actions
+
+import frc.team5499.frc2018Kotlin.auto.routines.Routine
+import frc.team5499.frc2018Kotlin.auto.routines.Routine.RoutineTag
+import frc.team5499.frc2018Kotlin.subsystems.Drivetrain
+
+public class DriveStraightAction(timeout: Double, inches: Double) : Action(timeout) {
+    
+    var mInches: Double
+
+    init{
+        mInches = inches
+    }
+
+    override public fun start() {
+        super.start()
+        Drivetrain.setPosition(mInches)
+    }
+
+    override public fun update() {
+        super.update()
+    }
+
+    override public fun next(): Boolean {

```

Comment:
this should return true if the drivetrain has reached the setpoint OR if its timed out

Reply:
in path pursuit, that's done with drivetrain.getPositionError(), but that isn't a method in the kotlin drivetrain. I'm probably missing something. Should I add it?

Reply:
oh, nevermind. It's a getter